test/format/value.test.ts

```diff
@@ -320,6 +320,24 @@ describe('format', function () {
                     },
                 },
             );
+            io.set(
+                {$type: 'yql.tztimestamp64', $value: 1, $optional: 2},
+                {
+                    json: {
+                        plain: 1,
+                        html: '<span class="yql_tztimestamp64">1</span>',
```

Comment:
It is better to have at least one test at least one test with a value greater than Number.MAX_SAFE_INTEGER  for each 64-bit type